tests/pstream.js

```diff
@@ -287,7 +287,7 @@ describe('PStream', () => {
   describe('sendMessage', () => {
     const callsid = 'testcallsid';
     const content = { foo: 'content' };
-    const messagetype = 'user-defined-message';
+    const messagetype = 'UserDefinedMessage';
```

Comment:
Let's keep the real value 'user-defined-message' in the test. Later, we'll add more types here.